apps/web/src/app/_components/search/search-filter.tsx

```diff
@@ -94,13 +106,16 @@ export default function SearchFilter({
     <Form {...form}>
       <form
         onSubmit={form.handleSubmit(onSubmit)}
-        className={cn("w-[100vw]", searchClassName)}
+        className={cn("w-[90vw]", searchClassName)}
```

Comment:
Right now the Clear button doesn't align with the right hand side of the screen like it does with the figma

<img width="1366" alt="Screenshot 2025-04-08 at 10 15 23 AM" src="https://github.com/user-attachments/assets/1d3dc764-f894-4469-9b2f-7687936a88a9" />

One solution I found (which also preserves the old styles) is 

```
      <form
        onSubmit={form.handleSubmit(onSubmit)}
        className={cn(
          "w-[98vw] border border-blue-700",
          searchType === "COMPANIES" && "w-full",
          searchClassName,
        )}
      >
```

Let me know if the `w-[98vw]` still gives you that weird issue we encountered where the width of the companies page is too much